docs/metrics.md

```diff
@@ -30,6 +33,12 @@ The number of nodes in the cluster that have the devices.kubevirt.io/kvm resourc
 ### kubevirt_number_of_vms
 The number of VMs in the cluster by namespace. Type: Gauge.
 
+### kubevirt_over_subscribed_container_rss_memory_bytes
```

Comment:
This name is too explicit. We monitor it not only when it's over subscribed.  Please update the name to address this. Something like : kubevirt_rss_and_requested_memory_diff_bytes

Reply:
maybe we should use `kubevirt_rss_to_requested_memory_max_diff_bytes`